hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -757,6 +759,14 @@ ppResult name (Report tests discards coverage result) = do
         ] ++
         ppCoverage tests coverage
 
+ppFailedAtLocation :: Maybe Span -> Doc Markup
+ppFailedAtLocation (Just span') =
+  "at" <+>
+  WL.text (spanFile span') <> ":" <>
+  WL.pretty (unLineNo (spanStartLine span')) <> ":" <>
+  WL.pretty (unColumnNo (spanStartColumn span'))
+ppFailedAtLocation Nothing = mempty
```

Comment:
Just a few [style](https://github.com/hedgehogqa/haskell-hedgehog/blob/master/STYLE_GUIDE.md#avoid-pattern-matching-and-guards-in-function-definitions) [nits](https://github.com/hedgehogqa/haskell-hedgehog/blob/master/STYLE_GUIDE.md#avoid-using-primes-when-naming-things), would probably make this:

```haskell
ppFailedAtLocation = \case
  Just x ->
    "at" <+>
    WL.text (spanFile x) <> ":" <>
    WL.pretty (unLineNo (spanStartLine x)) <> ":" <>
    WL.pretty (unColumnNo (spanStartColumn x))
  Nothing ->
    mempty
```

You could also use `span_` or `span0` but I think `x` is fine/neater because it's clear from usage what it is.

Reply:
Gotcha, thanks! I'll correct those.